lib/teamsnap/version.rb

```diff
@@ -1,3 +1,3 @@
 module TeamSnap
-  VERSION = "2.0.0.beta14"
+  VERSION = "2.0.0.beta15"
```

Comment:
I'd say it is time.  We've been in beta since March 17th 2016, well over a year.

https://github.com/teamsnap/teamsnap_rb/commit/f1a69a2e7fa28034950246a1b8188b0c347e2341